crates/agglayer-prover/src/lib.rs

```diff
@@ -51,9 +51,10 @@ pub fn main(cfg: PathBuf, version: &str, program: &'static [u8]) -> anyhow::Resu
 
     Ok(())
 }
+
 pub fn get_vkey(program: &'static [u8]) -> String {
     let vkey = prover_executor::Executor::get_vkey(program);
-    vkey.bytes32().to_string()
+    vkey.bytes32()
 }
```

Comment:
Is this function even used? I don't think it is in our codebase but I see it's public so external dependents may use it. Still I am not sure it provides much value.